python/ray/_raylet.pyx

```diff
@@ -702,3 +710,15 @@ cdef class CoreWorker:
         return ProfileEvent.make(
             self.core_worker.get().CreateProfileEvent(c_event_type),
             extra_data)
+
+    def deserialize_actor_handle(self, const c_string &bytes):
```

Comment:
Can we call this `deserialize_and_register_handle`? Otherwise, it is a little mysterious what this method does.

Reply:
Actually, why not defer the handle registration to when it is first used?

Reply:
Not sure what you mean by deferring the registration or why this is better. I think we'd have to cache the handle somewhere until it is used to do that, or do you mean something else?

Reply:
I think it's just weird to have special serialize/deserialize hooks, but if it's unavoidable it's fine...